### ISSUE TITLE
Concept page: Plugin callbacks are fired only after concept appendix has been resolved

### ISSUE THREAD

**Issue by kouralex** [NatLibFi/Skosmos#1091]
Whilst developing a new plugin, I encountered that plugins on concept page are displayed only after the mapped properties are resolved even though this is strictly not needed for most plugins, I suppose.

There exists the `skosmos:loadExternalResources` setting but it does not fully combat the issue as the Skosmos installation will still search for the possibly existing, known URIs (set in `config.ttl`) in AJAX style, which may be slow.

The source of the problem can be tracked down to the following code
https://github.com/NatLibFi/Skosmos/blob/5a7175ed01fdba9108dbf219680f2d4d4f9257fc/resource/js/docready.js#L290-L304
which is by the way duplicated (and should, thus, be unified) by https://github.com/NatLibFi/Skosmos/blob/5a7175ed01fdba9108dbf219680f2d4d4f9257fc/resource/js/docready.js#L1149-L1161

As one can guess, the loadMappingProperties() function will eventually call makeCallbacks(), as is evident by the code below:
https://github.com/NatLibFi/Skosmos/blob/5a7175ed01fdba9108dbf219680f2d4d4f9257fc/resource/js/scripts.js#L376-L451

It seems like there is a need for a separation of plugins that require information derived from the loaded mapped properties that must wait for the appendix to complete the AJAX query and for plugins that can immediately show their content. The behavior of each plugin should preferably be a plugin-dependent setting.

**Comment by osma** [comment]
This is implemented differently in Skosmos 3. Closing this issue. Please reopen if still relevant.
